pkgs/development/interpreters/cg3/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchurl
+, cmake
+, icu
+, boost
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cg3";
+  version = "1.3.7";
+
+  src = fetchurl {
+    url = "https://github.com/GrammarSoft/${pname}/releases/download/v${version}/${pname}-${version}.tar.bz2";
+    sha256 = "d8d996c60063baae2be1088d6122678179ab3c47653e76edd6cda07a423f4660";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  buildInputs = [
+    icu
+    boost
+  ];
+
+  # cmakeFlags = [
+  #   "-DCMAKE_INSTALL_PREFIX=${prefix}/"
+  # ];
+
+  # TODO: tcmalloc unless darwin?
+  # option(ENABLE_PYTHON_BINDINGS "Set to ON to build the Python wrapper" OFF)
+
+  meta = with lib; {
+    homepage = "https://github.com/GrammarSoft/cg3";
+    description = "Constraint Grammar interpreter, compiler and applicator vislcg3";
+    maintainers = [ lib.maintainers ];
+    license = licenses.gpl3;
```

Comment:
Files seem to use the gpl3plus version of the license (https://github.com/GrammarSoft/cg3/blob/990ac64551827508d32f78895a09b1ff331a4c24/src/ApertiumApplicator.cpp#L11)

Reply:
doh, fixing